### ISSUE TITLE
Suggestion: Implement report logger functionality

### ISSUE THREAD

**Issue by lehersabharwal** [picuscreative/testcafe-reporter-html#7]
Can we have a report logger where we can add the logs in our tests and the same would be reflected in the generated html report?

**Comment by davidcunha** [comment]
Hello @lehersabharwal !

Probably I didn't really got the issue. Can you describe a bit more what's the request?

Thanks! 😎

**Comment by lehersabharwal** [comment]
Sure. So currently, we can display logs or any information that we want to print by uding console.log in our tests.

Can we get a logger to display information in the generated html report as well.

e.g. we perform any assertion and then we want to print any message after that step. So when i view the report, the test cases can have that information as well in the report. 

@davidcunha

**Comment by davidcunha** [comment]
@lehersabharwal

Yes I got it now! One thing we gonna add in the near future is to display logs from the test itself while the test suite is running. This part is paramount for getting feedback about what's happening.

Regarding displaying console.log into the HTML tests, we don't have any plans to work on this. But feel free to fork it and submit a PR 😃

**Comment by alienintheheights** [comment]
Related to this, is there anyway to disable test output from being written to standard out when using this html reporter? It would be better (especially for our QA team) if the output was limited to report.html only. Great report tool, btw.

**Comment by davidcunha** [comment]
@alienintheheights Thanks for your input. We'll also check this possible feature. Anyway feel free to contribute and submit a PR!

**Comment by csheehyviz** [comment]
> @lehersabharwal
> 
> ... One thing we gonna add in the near future is to display logs from the test itself while the test suite is running. This part is paramount for getting feedback about what's happening.
> .....


I'm wondering if there is an update to this?
